### DISCUSSION TITLE
Translation of two WooCommerce-elements

### DISCUSSION THREAD

**Discussion by SebGbs** [bootscore/bootscore#744] [Q&A] [answered]
Hello!
I'm currently using the theme with WooCommerce and I'm having some trouble translating two elements:
"Cart"
"To find out your shipping cost, please proceed to checkout."
Both are placed in the **offcanvas-cart**.
I've added a new folder in the child theme called "languages" and added "da_DK.po" with the two translations, but it doesn't seem to work.
Can I do it this way, so I don't have to install Loco Translate?

**Comment by crftwrk** [comment, marked as answer]
Hello,

interesting question. In your case I suggest to override `template-parts/header/actions-woocommerce.php` and `/woocommerce/cart/mini-cart.php` in your child and roughly hard-code those strings. Then add or delete something in your cart to trigger AJAX.

At this point, you may interested to preview and test our upcoming v6 version which is closed to ship and has several new features/improvements in cart area and string handling.

- [Download](https://github.com/bootscore/bootscore/archive/refs/heads/v6.0.0-beta1.zip) bootscore v6 branch
  - Unzip and rename folder to `bootscore`
  - Zip and upload
- [Download](https://github.com/bootscore/bootscore-child/archive/refs/heads/v6.zip) child v6 branch
  - Unzip and rename folder as you like
  - Zip, upload and activate  

Use a filter in child's `functions.php` to change offcanvas cart title:

```php
/**
 * Change cart offcanvas title
 */
function change_cart_offcanvas_title($title) {
  return 'My Cart';
}
add_filter('bootscore/offcanvas/cart/title', 'change_cart_offcanvas_title');
```

"To find out your shipping cost, please proceed to checkout." has been deleted. Use a hook to drop a custom notice:

```php
/**
 * Offcanvas cart footer note
 */
add_action( 'woocommerce_widget_shopping_cart_before_buttons', 'bootscore_cart_footer_note' );

// Function for `woocommerce_widget_shopping_cart_before_buttons` action-hook.
function bootscore_cart_footer_note(){

  // action...
  echo '<div class="small text-body-secondary mb-3">Custom offcanvas cart footer notice</div>';

}
```

Add or remove something in your cart to trigger AJAX and view changes.

**Reply by SebGbs** [reply]
Hello @crftwrk 
Thanks for your reply - it worked :) I just needed to change "Cart" in **/template-parts/header/offcanvas-woocommerce.php** instead of **template-parts/header/actions-woocommerce.php**.
I think this solution will stand for now and I'll try the v6 out in future projects :)
